agr_api/src/main/java/org/alliancegenome/api/translators/tdf/DiseaseAnnotationToTdfTranslator.java

```diff
@@ -325,6 +325,19 @@ private static StringJoiner getStringJoiner(GeneDiseaseAnnotationDocument annota
 		return evidenceJoiner;
 	}
 
+	private static StringJoiner getStringJoiner(org.alliancegenome.curation_api.model.entities.DiseaseAnnotation annotation, Function<org.alliancegenome.curation_api.model.entities.ontology.ECOTerm, String> function) {
```

Comment:
@cmpich do we need the full package path to these classes?

Reply:
Unfortunately, yes. We have a DiseaseAnnotation class in the agr_java_software repo.